arch/arm/src/c5471/c5471_ethernet.c

```diff
@@ -1892,7 +1892,7 @@ static int c5471_ifup(struct net_driver_s *dev)
 
   putreg32((getreg32(EIM_CTRL) | EIM_CTRL_ESM_EN), EIM_CTRL);      /* enable ESM */
   putreg32((getreg32(ENET0_MODE) | ENET_MODE_ENABLE), ENET0_MODE); /* enable ENET */
-  up_mdelay(100);
+  nxsig_usleep(100000);
```

Comment:
The `up_mdelay` does not depend in scheduler configuration while `nxsig_usleep` does.

Reply:
What's the dependence you mean? the schedule tick?

Reply:
Yes